packages/manager/apps/catalog/src/components/SearchBar/SearchBar.tsx

```diff
@@ -67,6 +67,10 @@ const SearchBar: React.FC<SearchbarProps> = ({
     setLocalSearchValue(event.detail.value);
   };
 
+  useEffect(() => {
+    setSearchValue(localSearchValue);
```

Comment:
is it possible to do that on line 54 in the `useEffect` on `location.search` instead ? like remove the `if (searchURL)` or moving the set outside of this if statement

Reply:
Okay I will check the behaviour.